docs/dxp/7.x/en/site-building/creating-pages/creating-a-master-page-template.md

```diff
@@ -1,14 +1,14 @@
 # Creating a Master Page Template
 
-```note::
-  Master Page Templates are available since Liferay DXP 7.3.
-```
+> Liferay DXP 7.3+
+
+Custom Master Page Templates provide <!-- what does it mean that they "provide" common elements? does everyone know what that means? --> common elements, such as a Header and Footer Fragment.
```

Comment:
> what does it mean that they "provide" common elements? does everyone know what that means?

Header and Footer elements/fragments. I can clarify and say page elements that are common to every page (i.e. a Header and a Footer). They can really contain anything you like, but are primarily used for Headers and Footers at the moment.